src/app/pages/public/LifePage/index.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import BasicLayout from '../../../layouts/BasicLayout';
+import { useGetLifeLazyQuery } from '../../../api';
+import NotFoundResult from '../NotFoundPage';
+import LoadingElement from '../../../components/LoadingElement';
+import { Descriptions, Tag } from 'antd';
+
+const LifePage = () => {
+    const { id } = useParams();
+    console.log('id: ', id);
+    const [fetch, { data, loading }] = useGetLifeLazyQuery();
+
+    useEffect(() => {
+        fetch({ variables: { id } });
+    }, []);
+
+    if (loading) return <LoadingElement />;
+    if (!data) return <NotFoundResult />;
+    console.log('data: ', data);
```

Comment:
You should run your linter, most likely you didn't set it up with your editor as well, remember we use both `eslint` and `prettier`